include/libhal-arm-mcu/stm32f411/uart.hpp

```diff
@@ -0,0 +1,78 @@
+// Copyright 2024 Khalil Estell
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <cstdint>
+
+#include <libhal/initializers.hpp>
+#include <libhal/serial.hpp>
+
+#include "constants.hpp"
+#include "dma.hpp"
+#include "libhal-arm-mcu/stm32_generic/uart.hpp"
```

Comment:
```suggestion
#include <libhal-arm-mcu/stm32_generic/uart.hpp>
```

And move it up with the others.